### PR TITLE
Update UnitTestBot Python, JavaScript, Go docs

### PR DESCRIPTION
**Pull request by olganaumenko**
## Description

Linguistic bugs fixed, document structure optimized.
Added note: "be sure to enable _Experimental languages support_ option for **_each_** project".

## How to test

The PR introduces no code changes.
Manual scenario: the text itself has been previewed in the IntelliJ IDEA viewer.

## Self-check list

- [x] I've set the proper **labels** for my PR (at least, for category and component).
- [x] PR **title** and **description** are clear and intelligible.
- [ ] I've added enough **comments** to my code, particularly in hard-to-understand areas.
- [ ] The functionality I've repaired, changed or added is covered with **automated tests**.
- [x] **Manual tests** have been provided optionally.
- [ ] The **documentation** for the functionality I've been working on is up-to-date.

### CODE REVIEW

File: docs/JavaScriptSupport.md [hunk 1]
```diff
@@ -1,38 +1,42 @@
-# UnitTestBot JavaScript plugin setup
+# UnitTestBot JavaScript
```
Comment (rudolf101) [inline docs/JavaScriptSupport.md:1]: How about moving this file to utbot-js and renaming it to README, as is done with the Go and Python modules?

File: utbot-python/README.md [hunk 2]
```diff
@@ -1,48 +1,33 @@
...
+[UnitTestBot](https://www.utbot.org/) is the tool for automated unit test generation available as an IntelliJ IDEA plugin, or a command-line interface.
 
-This is the support of UTBot for Python.
+Now UnitTestBot provides fuzzing-based support for Python.
 
-UTBot tries to maximize the code coverage while minimizing the number of tests. For now, we use only the fuzzing technique for Python.
+## Requirements
 
-# Get started
+1. Python 3.8 or 3.9 (UnitTestBot Python has been tested to provide support for these versions; some Python 3.10 syntax is not supported)
```
Comment (tochilinak) [inline utbot-python/README.md:9]: Now we support Python 3.8+.

File: docs/JavaScriptSupport.md (hunk 1 above)
Comment (alisevych) [reply docs/JavaScriptSupport.md:1]: Probably moving Go- and Py- docs into /docs?

File: utbot-python/README.md [hunk 3]
```diff
@@ -1,48 +1,33 @@
...
 
-You can download both archives [here](https://github.com/UnitTestBot/UTBotJava/actions/runs/2956160534).
+_Note:_ the `com.intellij.modules.python` package should be specified in `/utbot-intellij/resources/plugin.xml` as a dependency for using Python PSI tree.
 
-## Python requirements
+## How to install and use
 
-UTBot Python has been tested on Python 3.8 and 3.9. Some syntax from Python 3.10 is not supported.
+To try UnitTestBot Python as an _**IntelliJ IDEA plugin**_:
+1. Download the corresponding [archive](https://github.com/UnitTestBot/UTBotJava/actions/runs/2956160534).
```
Comment (tochilinak) [inline utbot-python/README.md:20]: This instruction is outdated. Now UTBot Python is included in UTBot Java plugin.

File: docs/JavaScriptSupport.md (hunk 1 above)
Comment (rudolf101) [reply docs/JavaScriptSupport.md:1]: Both scenarios LGTM

File: utbot-go/README.md [hunk 4]
```diff
@@ -58,44 +58,45 @@ use the `go mod tidy` command. For editing and formatting `go.mod` files, use th
...
-### CLI
+## Command-line interface (CLI)
 
-#### Requirements
+### Requirements
 
 * Java SDK 11 or later
 * Go SDK 1.18 or later
 * Properly configured `go.mod` file for the code under test
 * GCC as well as `github.com/stretchr/testify/assert` and `golang.org/x/tools@v0.4.0` Go modules installed
```
Comment (egiptipavel) [inline utbot-go/README.md:91]: `golang.org/x/tools@v0.4.0` is not required

**Review by egiptipavel — APPROVED** [review]

**Review by rudolf101 — APPROVED** [review]

File: docs/PythonSupport.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+[UnitTestBot](https://www.utbot.org/) is the tool for automated unit test generation available as an IntelliJ IDEA plugin, or a command-line interface.
+
+Now UnitTestBot provides fuzzing-based support for Python.
+
+## Requirements
+
+1. Python 3.8 or later
+2. [Python plugin](https://plugins.jetbrains.com/plugin/631-python) for IntelliJ IDEA
+3. IntelliJ IDEA 2022.1
```
Comment (tochilinak) [inline docs/PythonSupport.md:11]: Now this line is not needed

File: docs/PythonSupport.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+## How to install and use
+
+To try UnitTestBot Python in your IntelliJ IDEA:
+1. To install the plugin, please refer to [UnitTestBot user guide](https://github.com/UnitTestBot/UTBotJava/wiki/Install-or-update-plugin).
+2. Configure the Python interpreter for your project and make sure IntelliJ IDEA resolves all the imports.
+3. In your IntelliJ IDEA, go to **File** > **Settings** > **Tools**, choose **UnitTestBot** and enable **Experimental languages support**.
+
+    **(!) NOTE:** be sure to enable this option for **_each_** project.
+
+4. To generate tests, place the caret at the required function and press **Alt+Shift+U** (**Alt+U, Alt+T** in Ubuntu).
```
Comment (alisevych) [inline docs/PythonSupport.md:26]: "in Ubuntu" -> better "in Linux", because there can be different distributives.

**Review by alisevych — CHANGES_REQUESTED** [review]
LGTM, just minor corrections are possible.

File: docs/JavaScriptSupport.md [hunk 7]
```diff
@@ -1,38 +1,42 @@
...
-* [Install](https://nodejs.org/en/download/) Node.js 10.0.0 or later
-* [Install](https://www.oracle.com/java/technologies/downloads/) Java 11 or later
-* Install _nyc_ 15.1.0 or later: `> npm install -g nyc`
-* Install Mocha 10.0.0 or later: `> npm install -g mocha`
+### Requirements
 
-## Basic usage
+* [Node.js 10.0.0 or later](https://nodejs.org/en/download/)
+* [Java 11 or later](https://www.oracle.com/java/technologies/downloads/)
+* _nyc_ 15.1.0 or later: `> npm install -g nyc`
```
[the remark this answers is not in the document]
Comment (rudolf101) [reply docs/JavaScriptSupport.md:38]: This is true, some npm packages are installed locally in each JS project, so an internet connection will be required at the first launch.

This only applies to npm packages.

File: docs/PythonSupport.md (hunk 6 above)
Comment (tamarinvs19) [reply docs/PythonSupport.md:26]: "in Linux" is too general restriction. The real problem is that keyboard shortcut `Alt+Shift` can be assigned to switch language and it blocks combination `Alt+Shift+U` but not all Linux users use `Alt+Shift` (e.g. me).

Comment (tochilinak) [reply docs/PythonSupport.md:26]: I think it it better to just write "or Alt+U, Alt+T". It works on Windows as well.

**Review by alisevych — APPROVED** [review]

**Pull request merged**